src/formatDate.js

```diff
@@ -50,7 +50,24 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArray = date.split(fromFormat[3]);
```

Comment:
it's better to create a variable for fromFormat[3] and toFormat[3], cause it's not obvious that they are separators